gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -179,11 +182,10 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         else:
             # Trigger mTLS. If the user overrides endpoint, use it as the mTLS
             # endpoint, otherwise use the default mTLS endpoint.
-            api_mtls_endpoint = (
-                (client_options.api_endpoint != _DEFAULT_ENDPOINT)
-                and client_options.api_endpoint
-                or _DEFAULT_MTLS_ENDPOINT
-            )
+            if client_options.api_endpoint != {{ service.client_name }}.DEFAULT_ENDPOINT:
+                api_mtls_endpoint = client_options.api_endpoint
+            else:
+                api_mtls_endpoint = {{ service.client_name }}.DEFAULT_MTLS_ENDPOINT
```

Comment:
Ah, I understand the logic now. This is fine as is, but what do you think about
```python
option_endpoint = client_options.api_endpoint
api_mtls_endpoint = self.DEFAULT_MTLS_ENDPOINT if option_endpoint == self.DEFAULT_ENDPOINT else option_endpoint
```
?

Judgement call on your part, I just like conditional assignment.

Reply:
yes, conditional assignment is better, applied your change